funsor/distributions.py

```diff
@@ -545,6 +545,48 @@ def eager_poisson(rate, value):
     return Poisson.eager_log_prob(rate=rate, value=value)
 
 
+class Gamma(Distribution):
```

Comment:
Could you factor these out as a separate PR and add simple tests?

Reply:
Done in #268